diff -ur libsodium-1.0.17-orig/configure.ac libsodium-1.0.17/configure.ac
--- libsodium-1.0.17-orig/configure.ac	2019-01-06 12:29:47.000000000 -0800
+++ libsodium-1.0.17/configure.ac	2021-05-17 18:22:25.582171400 -0700
@@ -15,7 +15,7 @@
 AC_SUBST(VERSION)
 
 SODIUM_LIBRARY_VERSION_MAJOR=10
-SODIUM_LIBRARY_VERSION_MINOR=2
+SODIUM_LIBRARY_VERSION_MINOR=0
 DLL_VERSION=23
 SODIUM_LIBRARY_VERSION=25:0:2
 #                       | | |
